app/models/starting_child.rb

```diff
@@ -23,7 +23,14 @@ def benefits_end
   # Return the next Monday if start_date is not nil.
   def adjusted_start_date
     return nil if start_date.nil?
-    start_date + (start_date.wday < 1 ? 1 : (1 - start_date.wday) + 7)
+
+    if start_date.wday < 1
+      start_date + 1
+    elsif start_date.wday > 1
+      start_date + ((1 - start_date.wday) + 7)
+    else
+      start_date
+    end
```

Comment:
@ikennaokpala Yeah, I think having days wrapping the whole expression reads strangely. 
It would be better to do `(1.days - start_date.wday) + 7.days` but then the days seems redundant.